index.js

```diff
@@ -80,10 +79,22 @@ module.exports = function (hljs) {
           },
         ],
       },
+      {
+        className: "attribute",
+        begin: "@",
+        end: "\\(",
+        excludeEnd: true,
+      },
       {
         className: "keyword",
         beginKeywords: KEYWORDS,
       },
+      {
+        // Attributes
+        className: "title",
+        begin: "\\b@[a-z][a-za-z0-9_]*\\b",
+        relevance: 0,
+      },
```

Comment:
Oh dear! This was a mistake on my part. This shouldn't be here, I've removed it from the website.

Reply:
I hope it is correct now!